phone/src/apps/camera/components/grid/GalleryGrid.tsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
-import { Box, Button } from '@material-ui/core';
+import { Box, Button, CircularProgress } from '@material-ui/core';
 import useStyles from './grid.styles';
-import { usePhotos } from '../../hooks/usePhotos';
+import { useCamera } from '../../hooks/useCamera';
```

Comment:
Remember to change the import in GalleryModal and ShareModal. Those are still trying to import 'usePhotos'